<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="psidot_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>psidot0 [rad/s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="theta0_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>theta0 [rad]</string>
   </property>
  </widget>
  <widget class="QLabel" name="R_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R [cm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="k_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>k [s/cm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a [cm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>m [g]</string>
   </property>
  </widget>
  <widget class="QLabel" name="TolConQual_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance Conserved Quality</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Friction">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Friction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="import_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TolConQual">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>10^-6</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="k">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="psidot0">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>250</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="theta0">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
